core/src/main/java/io/confluent/kafka/schemaregistry/masterelector/kafka/SchemaRegistryCoordinator.java

```diff
@@ -67,18 +68,20 @@ public SchemaRegistryCoordinator(
       long retryBackoffMs,
       SchemaRegistryIdentity identity,
       SchemaRegistryRebalanceListener listener) {
-    super(logContext,
+    super(new GroupRebalanceConfig(
+            sessionTimeoutMs,
+            rebalanceTimeoutMs,
+            heartbeatIntervalMs,
+            groupId,
+            Optional.empty(),
+            retryBackoffMs,
+            true
+                    ),
```

Comment:
this should be in line above at least, probably

Reply:
Oops! Sorry about that.